There is a test that change the results depending on the tests performed beforehand

```
$ python -m pytest -x cupyx_tests/fallback_mode_tests/test_notifications.py
======================================= test session starts ========================================
platform linux -- Python 3.6.8, pytest-5.3.2, py-1.8.1, pluggy-0.13.1
rootdir: /home/anaruse/work/CUPY/cupy, inifile: setup.cfg
collected 9 items                                                                                  

cupyx_tests/fallback_mode_tests/test_notifications.py .........                              [100%]

======================================== 9 passed in 1.77s =========================================
```

```
$ python -m pytest -x cupyx_tests/fallback_mode_tests/test_notifications.py -k test_seterr
======================================= test session starts ========================================
platform linux -- Python 3.6.8, pytest-5.3.2, py-1.8.1, pluggy-0.13.1
rootdir: /home/anaruse/work/CUPY/cupy, inifile: setup.cfg
collected 9 items / 8 deselected / 1 selected                                                      

cupyx_tests/fallback_mode_tests/test_notifications.py F                                      [100%]

============================================= FAILURES =============================================
_______________________________ TestNotifications.test_seterr_geterr _______________________________

self = <cupyx_tests.fallback_mode_tests.test_notifications.TestNotifications testMethod=test_seterr_geterr>

    def test_seterr_geterr(self):
    
        default = _ufunc_config.geterr()
>       assert default['fallback_mode'] == 'warn'
E       AssertionError: assert 'ignore' == 'warn'
E         - ignore
E         + warn

cupyx_tests/fallback_mode_tests/test_notifications.py:20: AssertionError
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! stopping after 1 failures !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
================================= 1 failed, 8 deselected in 0.38s ==================================
```

```
$ python -c "import cupy; cupy.show_config()"
CuPy Version          : 8.0.0rc1
CUDA Root             : /usr/local/cuda-11.0.2
CUDA Build Version    : 11000
CUDA Driver Version   : 11000
CUDA Runtime Version  : 11000
cuBLAS Version        : 11100
cuFFT Version         : 10201
cuRAND Version        : 10201
cuSOLVER Version      : (10, 6, 0)
cuSPARSE Version      : 11101
NVRTC Version         : (11, 0)
Thrust Version        : 100909
CUB Build Version     : 100909
cuDNN Build Version   : None
cuDNN Version         : None
NCCL Build Version    : None
NCCL Runtime Version  : None
cuTENSOR Version      : None
```

--- a/tests/cupyx_tests/fallback_mode_tests/test_notifications.py
+++ b/tests/cupyx_tests/fallback_mode_tests/test_notifications.py
@@ -11,8 +11,17 @@ from cupyx import _ufunc_config
 from cupyx_tests.fallback_mode_tests import test_fallback as test_utils
 
 
+class NotificationTestBase(unittest.TestCase):
+
+    def setUp(self):
+        self.old_config = _ufunc_config._config
+
+    def tearDown(self):
+        _ufunc_config.config = self.old_config
+
+
 @testing.gpu
-class TestNotifications(unittest.TestCase):
+class TestNotifications(NotificationTestBase):
 
     def test_seterr_geterr(self):
 
@@ -43,7 +52,7 @@ class TestNotifications(unittest.TestCase):
     {'func': fallback_mode.numpy.array_equiv, 'shape': (3, 4)},
 )
 @testing.gpu
-class TestNotificationModes(unittest.TestCase):
+class TestNotificationModes(NotificationTestBase):
 
     def test_notification_ignore(self):
 
@@ -98,7 +107,7 @@ class TestNotificationModes(unittest.TestCase):
 
 
 @testing.gpu
-class TestNotificationVectorize(unittest.TestCase):
+class TestNotificationVectorize(NotificationTestBase):
 
     @test_utils.enable_slice_copy
     def test_custom_or_builtin_pyfunc(self):

